documents/charter.md

```diff
@@ -15,7 +15,7 @@ Instead, it would be beneficial, for both the application provider and network o
 The application provider has the ability to measure end user QoE and therefore can self-adapt with QoE feedback.
 
 The Secure Communication of Network Properties (SCONEPRO) Working Group's primary objective is to specify an on-path protocol for securely communicating network properties to clients relevant to a given application, such as the maximum achievable bandwidth for a video. 
-- The working group will initially focus on a solution that communicates the maximum achievable bandwidth for a video using QUIC connections carrying those applications.
+- The working group will initially focus on a solution that communicates the maximum achievable bandwidth for a video delivered from a server to a client, using QUIC connections carrying the application signaling traffic.
```

Comment:
> using QUIC connections carrying the application **signaling traffic**.

I'm sure that we haven't been clear enough in writing, but I do think at least some of us have assumed that we're using QUIC for video transport, as well as for signaling transport. Am I the only one who thought that? 

If one of the BOF questions is "is SCONE worth doing", and video is carried by (say) HTTP/2 over TCP, is knowing available path bandwidth valuable enough to justify implementing and deploying SCONE?